README.md

```diff
@@ -1,5 +1,4 @@
-# Slack plugin for Jenkins
-
+# Slack plugin for Jenkins [![Build Status](https://travis-ci.org/jenkinsci/slack-plugin.svg?branch=master)](https://travis-ci.org/jenkinsci/slack-plugin)
```

Comment:
Assuming you still have the `upstream` remote configured, might I suggest the following.

```bash
git checkout upstream -- README.md
git add README.md
git commit --amend
git push origin --force
```